### DISCUSSION TITLE
SD card deck

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1238] [Q&A]
Hi, Can I use SD card deck to record the UWB anchor data ?

**Comment by knmcguire** [comment]
Sure as long as the crazyflie is able to receive these data from the UWB anchors it should be able to record it. The deck can not be used for the loco node itself.

See the product page of the SD card deck for instructions of how to use it https://www.bitcraze.io/products/micro-sd-card-deck/
